packages/core/admin/admin/src/content-manager/pages/ListView/components/TableRows/index.js

```diff
@@ -178,19 +179,36 @@ export const TableRows = ({
                 );
               }
 
-              if (hasReviewWorkflows && name === 'strapi_reviewWorkflows_stage') {
-                return (
-                  <Td key={key}>
-                    {data.strapi_reviewWorkflows_stage ? (
-                      <ReviewWorkflowsStage
-                        color={data.strapi_reviewWorkflows_stage.color}
-                        name={data.strapi_reviewWorkflows_stage.name}
-                      />
-                    ) : (
-                      <Typography textColor="neutral800">-</Typography>
-                    )}
-                  </Td>
-                );
+              if (hasReviewWorkflows) {
+                if (name === 'strapi_reviewWorkflows_stage') {
```

Comment:
I'm starting to question why we haven't removed `reviewWorkflows` from the key here. Do you remember that?

Reply:
Yeah sure :) I'll make sure we unify this in the multiple workflows branch.